test/OptionSettlementEngine.integration.t.sol

```diff
@@ -259,7 +259,7 @@ contract OptionSettlementIntegrationTest is BaseEngineTest {
             totalExercised += claimData.amountExercised;
         }
 
-        assertEq(totalExercised, bucketsCreated * 1e18 - 1);
```

Comment:
why is this changing?

Reply:
https://github.com/valorem-labs-inc/valorem-core/blame/audit-fixes-2/test/OptionSettlementEngine.integration.t.sol
https://github.com/valorem-labs-inc/valorem-core/commit/5573d198898f4dd7c623c645af5b721c7b5caa48